siddhi-extension-archetype/siddhi-io/src/main/resources/archetype-resources/component/src/main/java/__typeOf_IOinLowerCase__/source/__classNameOfSource__.java

```diff
@@ -0,0 +1,179 @@
+package ${package}.${typeOf_IOinLowerCase}.source;
+
+import org.wso2.siddhi.annotation.Example;
+import org.wso2.siddhi.annotation.Extension;
+import org.wso2.siddhi.core.config.SiddhiAppContext;
+import org.wso2.siddhi.core.exception.ConnectionUnavailableException;
+import org.wso2.siddhi.core.stream.input.source.Source;
+import org.wso2.siddhi.core.stream.input.source.SourceEventListener;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+import org.wso2.siddhi.core.util.transport.OptionHolder;
+
+import java.util.Map;
+
+/**
+ * This is a sample class-level comment, explaining what the extension class does.
+ */
+
+/**
+ * Annotation of Siddhi Extension.
+ * <pre><code>
+ * eg:-
+ * {@literal @}Extension(
+ * name = "The name of the extension",
+ * namespace = "The namespace of the extension",
+ * description = "The description of the extension (optional).",
+ * //Sink configurations
```

Comment:
this is not sink. Check all other places where this comment is used

Reply:
Fixed in 624c01a